tests/phpunit/tests/test-model.php

```diff
@@ -235,6 +232,8 @@ public function test_maybe_create_language_terms() {
 		}
 
 		self::$model->clean_languages_cache();
+		$links_model = self::$model->get_links_model();
+		$GLOBALS['polylang'] = new PLL_Admin( $links_model );
```

Comment:
This is here only to validate taxonomies right?
If not set, does the validation still occur?

Reply:
It used in `Options\Business\Language_Taxonomies` (with `PLL()`).